edx-olive/edx-platform
# CAM2-353 Recompiles the translations for LMS and CMS

Need to compile both LMS and CMS JS translation changes together to get the static asset hash to match and prevent JS loading errors.

Compiled using:

```
django-admin.py compilemessages -v0
./manage.py lms compilejsi18n --settings=devstack
./manage.py cms compilejsi18n --settings=devstack
```

**Testing instructions**

Will deploy on stage.

1. Change your language to Hebrew.
1. Ensure that you can load the course view in Studio
1. Ensure that you can load the course view in LMS.

**Reviewer**
